### PR TITLE
Fix typo in comment

### PR DESCRIPTION
**Pull request by MVrachev**
Signed-off-by: Martin Vrachev <mvrachev@vmware.com>

### CODE REVIEW

**Review by trishankatdatadog — APPROVED** [review]

File: tuf/repository_lib.py [hunk 1]
```diff
@@ -1476,7 +1476,7 @@ def generate_snapshot_metadata(metadata_directory, version, expiration_date,
   metadata_directory = _check_directory(metadata_directory)
 
   # Snapshot's 'fileinfodict' shall contain the version number of Root,
-  # Targets, and all delegated roles fo the repository.
+  # Targets, and all delegated roles to the repository.
```
Comment (lukpueh) [inline tuf/repository_lib.py]: Wouldn't "of" make more sense?

Comment (MVrachev) [reply tuf/repository_lib.py]: Yes, a good point.
I am not a native english speaker and sometimes I make mistakes like that one. :D

Comment (lukpueh) [reply tuf/repository_lib.py]: That makes two of use. ;)

Comment (trishankatdatadog) [reply tuf/repository_lib.py]: Oh, no, me three...

**Comment by MVrachev** [comment]
Squashed my commits and rebased on develop.

**Review by lukpueh — APPROVED** [review]
Merci! :)

**Pull request merged**